documentation/docs/04-hooks.md

```diff
@@ -116,3 +116,23 @@ export async function serverFetch(request) {
 	return fetch(request);
 }
 ```
+
+### handleError
+
+This function allows you to handle errors with your own error handler.
+
+For example, you may have a custom logging scheme that you send to an external
+system, and you want to format the error in a manner consistent with your other
+logs.
+
+```js
+/** @type {import('@sveltejs/kit').HandleError} */
+export async function handleError(error) {
```

Comment:
I'd consider making the input an object with a field named `error`. We've done that as a general rule for backwards compatibility. E.g. it might be helpful to later add the request since someone might want to log what URL the error occurred on

```suggestion
export async function handleError({ error }) {
```

Reply:
Makes sense, I like that.